js/app.js

```diff
@@ -92,6 +127,9 @@
    * to set the context to the correct object (the current context)
    */
 
+   btn6.addEventListener(CLICK, function() {
+    setMyLightGreen.call(btn6);
```

Comment:
this one is asking for the current context. Replace "btn6" with the current context